### PR TITLE
At a glance: add Activity Log card

### LINKED ISSUE #8132: Activity Log - Add Activity Dashitem to Jetpack dashboard
This is a bare bones attempt to add a gateway to the activity log into the Jetpack dashboard. It is simple out of respect for the limited time and because I don't want to have use refactor the Jetpack Dashboard (yet).

#### Site with backups

activity - with backups

#### Site without backups

activity - without backups

About the only difference between the two is a link to upgrade. You'll also notice I rearranged the other dashitems a bit to be more cohesive in order.

### PR DESCRIPTION
**Pull request by eliorivero**
Fixes #8132

#### Changes proposed in this Pull Request:

* introduces a new card Activity to point user to Activity Log

#### Dev Mode
captura de pantalla 2017-11-17 a la s 11 00 28

#### Free, Personal and Premium plans
captura de pantalla 2017-11-20 a la s 18 44 19

#### Professional plan
captura de pantalla 2017-11-17 a la s 10 58 13

#### Testing instructions:

* test this with a Professional plan and without it

### CODE REVIEW

**Comment by oskosk** [comment]
Will review but...

We're trying to introduce prop-types in this one #8158

File: _inc/client/at-a-glance/index.jsx [hunk 1]
```diff
@@ -77,29 +78,33 @@ class AtAGlance extends Component {
 					}
 					<div className="jp-at-a-glance__item-grid">
 						<div className="jp-at-a-glance__left">
-							<DashProtect { ...settingsProps } />
+							<DashActivity { ...settingsProps } />
 						</div>
 						<div className="jp-at-a-glance__right">
-							<DashScan { ...settingsProps } siteRawUrl={ this.props.siteRawUrl } />
+							<DashProtect { ...settingsProps } />
```
Comment (oskosk) [inline _inc/client/at-a-glance/index.jsx]: Looks like `this.props.siteRawUrl` is empty (`""` ) at this point. Probably it wasn't being used and/or passed down from its parent.

**Comment by MichaelArestad** [comment]
@eliorivero the other cards show an Upgrade notice when they are not available on the current plan. Can you take a look at how those work and try an upgrade notice for folks on the JP free plan?

**Comment by eliorivero** [comment]
Definitely. I didn't add this because it wasn't in the mockup but I'll add it now.

**Comment by MichaelArestad** [comment]
@eliorivero yeah that was my fault. I forgot about those until I started switching around plan types.

File: _inc/client/at-a-glance/index.jsx (hunk 1 above)
Comment (eliorivero) [reply _inc/client/at-a-glance/index.jsx]: That's odd @oskosk I'm seeing this defined at all times. I added a console.log at the beginning of render() and it shows the function perfectly.

**Comment by eliorivero** [comment]
@MichaelArestad it's now updated 👍 I've updated the corresponding screenshot in this PR too.

File: _inc/client/at-a-glance/index.jsx (hunk 1 above)
Comment (oskosk) [reply _inc/client/at-a-glance/index.jsx]: weird...

**Comment by MichaelArestad** [comment]
Looks good! 🚢

**Comment by oskosk** [comment]
Tested in another environment. Still keep getting `siteRawUrl` as an empty string . This time I tested with a fresh installation of WordPress and this PR using the Jetpack Beta Tester Plugin.

image


image

**Comment by oskosk** [comment]
I added a commit fixing the above ^.

**Comment by eliorivero** [comment]
Thanks @oskosk! updated it to pass only `siteRawUrl`.

**Comment by oskosk** [comment]
I'm seeing the card like this with Premium and Personal plans:

image

Note the *Active* label.

**Pull request merged**